src/components/structures/UserMenu.tsx

```diff
@@ -170,10 +169,6 @@ export default class UserMenu extends React.Component<IProps, IState> {
         return SettingsStore.getValue("doNotDisturb");
     }
 
-    private get hasHomePage(): boolean {
-        return !!getHomePageUrl(SdkConfig.get());
-    }
```

Comment:
This could use a comment on the diff with some context on why it's not necessary anymore.

Ex. We always want the option to go to home. A homepage URL isn't necessary for the welcome view to exist. By default it will display the `Welcome to Element` screen (with a screenshot of that).

Reply:
We want to provide the user the option to go home at any time. This allows the user to "close" the room they were reading so they can walk away or perform another task on their computer and not accidentally have the room marked as read when they return to Element.

When there is no custom home page URL, clicking the home button will bring the user back to the Welcome to BRAND page:
<img width="978" src="https://user-images.githubusercontent.com/5855073/167056695-15a48830-0345-4cfd-8ca7-3c817b6bafb3.png">

When a custom home page URL is set, clicking home will bring the user to that custom page.

(I'm not totally sure what you mean by on the diff. In the code or just in this thread?)

Reply:
> (I'm not totally sure what you mean by on the diff. In the code or just in this thread?)

@aaronraimist This is perfect 👌. Could have been a new thread on the diff too.

Thanks for the explaining the two flows 🙂

Reply:
I also added this information to the issue description so it's easy to understand what this PR does ⏩

Looks like a nice little change to me 👍